stable/horovod/templates/config.yaml

```diff
@@ -1,8 +1,8 @@
 {{- $workerNum := .Values.worker.number -}}
 {{- $name := include "horovod.fullname" . }}
 {{- $slots := 1 }}
-{{- if index .Values.resources "nvidia.com/gpu" }}
-{{- $slots := index .Values.resources "nvidia.com/gpu" }}
+{{- if index .Values.resources.requests "nvidia.com/gpu" }}
```

Comment:
I1007 06:26:50.183] [ERROR] templates/: render error in "horovod/templates/config.yaml": template: horovod/templates/config.yaml:4:7: executing "horovod/templates/config.yaml" at <index .Values.resour...>: error calling index: index of untyped nil

Reply:
I fixed this.